.github/workflows/dotnet-instr-deployer-add-on.yml

```diff
@@ -4,13 +4,27 @@ on:
   push:
     paths:
       - 'packaging/dotnet-instr-deployer-add-on/**'
+      - '.github/workflows/dotnet-instr-deployer-add-on.yml'
   pull_request:
     paths:
       - 'packaging/dotnet-instr-deployer-add-on/**'
+      - '.github/workflows/dotnet-instr-deployer-add-on.yml'
   workflow_dispatch:
+    inputs:
```

Comment:
I assume these versions will have to be updated manually, is that correct? Is there anyway to update at some frequency?

Reply:
Unfortunately, I couldn't find a way to get the latest version number and hash via some "latest" or similar mechanism.